platform/docs/docs/platform/services/ui/customization-service.md

```diff
@@ -275,6 +275,50 @@ const overlayItem: Types.UIOverlayItem = {
 
 This section can be used to specify various customization capabilities.
 
+## Context Menus
+The context menu customization is a relatively complex one, with two layers.
+The first layer is the choice of the command to run to display the context menu.
+For the case of the cornerstone viewer context menu, this is specified by a
+customization called `showViewerContextMenu`.  Then, the actual menu is chosen
+by the menuName as specified in the showViewerContextMenu, OR by the actual
+items list in thew viewer context menu.
```

Comment:
This is confusing to me, what do we mean this is `specified` by a customization called `showViewerContextMenu`? I don't understand this part